aiidalab_widgets_base/computational_resources.py

```diff
@@ -937,7 +970,7 @@ def _reset(self):
         self.label.value = ""
         self.hostname.value = ""
         self.description.value = ""
-        self.work_dir.value = ""
+        self.work_dir.value = "/home/{username}/aiida_run"
```

Comment:
work_dir cannot be empty. Should we provide some default value like this or not??

CC @yakutovicha @unkcpz

Reply:
I decided to revert this change. I think this needs to be a conscious and informed decision from the  user. On some HPCs, setting this to home directory could potentially lead to excessive network communication.